include/deal.II/fe/mapping_fe_field.h

```diff
@@ -582,30 +582,35 @@ class MappingFEField : public Mapping<dim,spacedim>
 
 private:
   /**
-   * Update internal degrees of
-   * freedom. */
+   * Update internal degrees of freedom.
+   */
   void update_internal_dofs(const typename Triangulation<dim,spacedim>::cell_iterator &cell) const;
 
+  /**
+   * Typedef holding the local cell_dofs.
+   */
+  typedef
+  Threads::ThreadLocalStorage <std::vector<double> > local_dofs_t;
+
+  /**
+  * Typedef holding the local dof indices.
+  */
+  typedef
+  Threads::ThreadLocalStorage <std::vector<types::global_dof_index> > local_dof_indices_t;
 
   /**
    * It stores the local degrees of freedom of the DH for each cell
    * (i.e. euler_vector * dof_indices, see method update_internal_dofs for more
    * clarifications.).
    */
-  mutable std::vector<double> local_dofs;
+  mutable local_dofs_t local_dofs;
```

Comment:
I know this is pre-existing, but it would also be good to rename this variable to `local_dof_values`.